android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -526,6 +528,22 @@ public boolean handleMessage(Message msg)
 				Activity activity = getWindowActivity();
 				if (activity != null) {
 					activity.setTitle(TiConvert.toString((Object) (msg.obj), ""));
+
+					if (windowActivity != null && windowActivity.get() != null
+						&& windowActivity.get().getSupportActionBar() != null) {
+						ActionBar actionBar = windowActivity.get().getSupportActionBar();
+						if (actionBar.getTitle() instanceof SpannableStringBuilder) {
+							SpannableStringBuilder stringBuilder;
+							stringBuilder = new SpannableStringBuilder(TiConvert.toString((Object) (msg.obj), ""));
+							if (barColor != -1) {
+								stringBuilder.setSpan(new ForegroundColorSpan(barColor),
+									0, stringBuilder.length(), Spannable.SPAN_INCLUSIVE_INCLUSIVE);
+							}
+							actionBar.setTitle(stringBuilder);
+						} else {
+							actionBar.setTitle(TiConvert.toString((Object) (msg.obj), ""));
```

Comment:
Which type is `msg.object` by default?

Reply:
It should be `Object` already. I used the same lines as the current code:
https://github.com/tidev/titanium-sdk/blob/master/android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java#L528

Android Studio has a hint `Casting '(msg.obj)' to 'Object' is redundant` but I left it there just in case.